app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/UpdateActivity.java

```diff
@@ -191,7 +191,7 @@ public static void checkForAnUpdate(final Context context) {
 
     public static void forceUpdateCheckNow() {
         JoH.static_toast_long(xdrip.gs(R.string.checking_for_update));
-        UpdateActivity.last_check_time = -2;
+        UpdateActivity.last_check_time = -1;
```

Comment:
What's the rationale behind this change? Does `-1` have a special meaning?

Reply:
The code is a little convoluted, but it basically interacts with this existing check here:
![image](https://user-images.githubusercontent.com/52087143/160037812-f82fc0a5-6fd8-4d81-8b3a-00cd2e05b72f.png)
If I had to translate it into pseudocode, I'd probably describe it as:
Do we skip the update check?
- If the last_check_time is set to -1, _never_ skip.
- If it's anything else, **SKIP** the check if the "automatic updates" switch is turned **OFF**


The function forceUpdateCheckNow() _is_ used in one other place (a function called `testAndSetNightly(..)`), which now won't care if "automatic updates" are switched off. I've been considering tidying up this wider logic before I formally submit this PR for review.